src/extension/options-page/InitStep.ts

```diff
@@ -1,7 +1,9 @@
 export enum InitStep {
-    'Setup1' = '1s',
-    'Setup2' = '2s',
-    'Restore1' = '1r',
-    'RestoreAdvanced1' = '1ra',
-    'Restore2' = '2r',
+    Disclaimer = '1d',
```

Comment:
the number "0" means it is the 0th step. the after char "s" or "r" is mean the different branch of the `n`th step.

If you think the old naming logic is not descriptive, feel free to move the naming style to `"step-0-disclaimer"`

Reply:
Temporarily naming it as `-1s` and the old initialization implementation will be replaced totally in [feature/dashboard](https://github.com/DimensionDev/Maskbook/pull/1026).